TAScheduler/Managment/UserManagement.py

```diff
@@ -28,28 +62,57 @@ def createUser(self):
     # User Address(in) - Address of the user
     # User Password(in) - Password of the user
     # User Type(in) - Type of the user
-    def editUser(self):
-        pass
+    def editUser(self, Id, name, contact, email, address, password, usertype):
+        if(not(isinstance(Id, int))):
+            raise TypeError("Id entered is not of type int")
+        if(not(isinstance(name, str))):
+            raise TypeError("Name entered is not of type str")
+        if(not(isinstance(contact, str))):
+            raise TypeError("Contact entered is not of type str")
+        if(not(isinstance(email, str))):
+            raise TypeError("Email entered is not of type str")
+        if(not(isinstance(address, str))):
+            raise TypeError("Address entered is not of type str")
+        if(not(isinstance(password, str))):
+            raise TypeError("Address entered is not of type str")
+        if(not(isinstance(usertype, int))):
+            raise TypeError("SSN entered is not of type int")
 
-    # Preconditions: The user has to have been instantiated
-    # The user must be of type administrator
-    # Postconditions: User is deleted
-    # Side-effects: User is deleted so it is removed from the database
-    # UserId(in) - Id of the user
-    def deleteUser(self):
-        pass
+        try:
+            self.findUser(Id)
+        except TypeError: 
+             raise TypeError("This user already exists(editUser): ")
 
     # Preconditions: The user has to have been instantiated
     # There are accounts to display
     # Postconditions: Displays the user
     # Side-effects: None
     # UserId(in) - Id of the user
-    def findUser(self):
+    def findUser(self, Id):
+        if(not(isinstance(Id, int))):
+            raise TypeError("Id entered is not of type int")
+        try:
+            profile = UserProfile.objects.get(userID = Id)
+        except UserProfile.DoesNotExist:
+            profile = None
+
+        if(profile == None):
+            raise TypeError("This ID does not exist")
+
+        return profile
+    # Preconditions: The user has to have been instantiated
+    # The user must be of type administrator
+    # Postconditions: User is deleted
+    # Side-effects: User is deleted so it is removed from the database
+    # UserId(in) - Id of the user
+    def deleteUser(self, Id):
+        profile = self.findUser(Id)
```

Comment:
No reason for this to be two lines.
```suggestion
        UserProfile.objects.delete(self.findUser(Id)
```